doc/sphinx/source/changelog.rst

```diff
@@ -3,6 +3,112 @@
 Changelog
 =========
 
+.. _changelog-v2-11-0:
+
+v2.11.0
+-------
+Highlights
+
+- New CMORizers have been added:
+
+  - Updating ESACCI-WATERVAPOUR cmorizer (:pull:`3282`) by :user:`malininae`
+  - CMORiser for ANU Climate 2.0 Australian data (:pull:`3511`) by :user:`flicj191`
+
+- NEW TREND: First time release manager shout-outs!
+
+  - This is the first ESMValTool release managed by the Met Office! We want to
+    shout this out - and for all future first time release managers to
+    shout-out - to celebrate the growing, thriving ESMValTool community.
+
+This release includes
+
+Bug fixes
+~~~~~~~~~
+
+-  Update OceanSODA-ETHZ CMORizer with new source file (:pull:`3535`) by :user:`TomasTorsvik`
```

Comment:
This entry looks like it would be better placed under `Observational and re-analysis dataset support`

Reply:
Done in 943e713d7 😊